ext/intl/config.m4

```diff
@@ -6,9 +6,25 @@ PHP_ARG_ENABLE(intl, whether to enable internationalization support,
 [  --enable-intl           Enable internationalization support])
 
 if test "$PHP_INTL" != "no"; then
-  PHP_SETUP_ICU(INTL_SHARED_LIBADD)
+  PKG_CHECK_MODULES([ICU], [icu-io >= 50.1],
+                    [ICU_FOUND=true], [ICU_FOUND=false])
+
+  if test "$ICU_FOUND" = false; then
```

Comment:
I think it would be better to drop the ICU_FOUND assignments and let PKG_CHECK_MODULES emit the default error message if icu is not found.

Reply:
Okay. Fine by me.